packages/common/base/src/types/react/nftDisplay.ts

```diff
@@ -0,0 +1,14 @@
+import { NFTOrNFTLocator, Wallet } from "@/blockchain/types";
+
+import { UIConfig } from "../uiconfig";
+
+interface CommonProps {
+    uiConfig?: UIConfig;
+    environment?: string;
+}
+export interface NFTCollectionViewProps extends CommonProps {
+    wallets: Wallet[];
+}
+export interface NFTDetailProps extends CommonProps {
+    nft: NFTOrNFTLocator;
+}
```

Comment:
re: `NFTOrNFTLocator` – if this isn't used I'd prefer to omit

Reply:
unfortunately we cant remove it